packages/dapp/src/components/GrantHeader.tsx

```diff
@@ -32,14 +32,15 @@ export const GrantHeader: React.FC<Props> = ({ grant }) => {
     >
       <Link
         color="white"
-        fontSize={{ base: '2rem', md: '3rem' }}
+        fontSize={{ base: '1.5rem', md: '3rem' }}
         fontWeight="800"
         textAlign="center"
         to={`/grant/${grant.id}`}
+        style={{ maxWidth: '600px', whiteSpace: 'pre-wrap' }}
       >
         {grant.name}
       </Link>
-      <Text>{grant.description}</Text>
+      <Text style={{ maxWidth: '600px' }}>{grant.description}</Text>
```

Comment:
in both places you can set `maxWidth="35rem"` directly as props. no need to set styles. 
if you are using vscode it will show you the different prop options.
https://next.chakra-ui.com/docs/features/style-props